lib/Doctrine/Common/Cache/PhpFileCache.php

```diff
@@ -106,4 +98,23 @@ protected function doSave($id, $data, $lifeTime = 0)
 
         return $this->writeFile($filename, $code);
     }
+
+    /**
+     * @param string $id
+     *
+     * @return array|false
+     */
+    private function includeFileForId($id)
+    {
+        $fileName = $this->getFilename($id);
+
+        // note: error suppression is still faster than `file_exists`, `is_file` and `is_readable`
+        $value = @include $fileName;
```

Comment:
Oh wow, I didn't expect this error suppression to be faster than using a temporary error handler either: http://3v4l.org/RFlvO. Its faster in almost every version. Interesting.